tests/integration/targets/inventory_flightctl/.config/flightctl/flightctl.inventory.yaml

```
@@ -0,0 +1,27 @@
+    field_selectors:
+      - metadata.owner = "Fleet/fleet-dev"
+  - name: test_devices
+    field_selectors:
+      - metadata.owner = "Fleet/fleet-test"
+  - name: unmanaged_devices
+    field_selectors:
+      - metadata.owner = null
+  - name: device_specific_group
+    field_selectors:
+      - metadata.name in ('ansible-integration-test-device-label-1', 'device-dev-02')
+request_timeout: 120
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)